aspnetcore/blazor/get-started.md

```diff
@@ -28,19 +28,20 @@ Get started with Blazor:
 
    1\. Install the latest [Visual Studio preview](https://visualstudio.com/vs/preview) with the **ASP.NET and web development** workload.
 
-   2\. Install the latest [Blazor extension](https://go.microsoft.com/fwlink/?linkid=870389) from the Visual Studio Marketplace. This step makes Blazor templates available to Visual Studio.
+   2\. Create a new project.
 
-   3\. Create a new project.
+   3\. Select **ASP.NET Core Web Application**. Select **Next**.
 
-   4\. Select **ASP.NET Core Web Application**. Select **Next**.
+   4\. Provide a project name in the **Project name** field or accept the default project name. Confirm the **Location** entry is correct or provide a location for the project. Select **Create**.
 
-   5\. Provide a project name in the **Project name** field or accept the default project name. Confirm the **Location** entry is correct or provide a location for the project. Select **Create**.
+   5\. In the **Create a new ASP.NET Core Web Application** dialog, confirm that **.NET Core** and **ASP.NET Core 3.0** are selected.
 
-   6\. In the **Create a new ASP.NET Core Web Application** dialog, confirm that **.NET Core** and **ASP.NET Core 3.0** are selected.
+   6\. For a Blazor client-side experience, choose the **Blazor WebAssembly App** template. For a Blazor server-side experience, choose the **Blazor Server App** template. Select **Create**. For information on the two Blazor hosting models, server-side and client-side, see <xref:blazor/hosting-models>.
```

Comment:
The client-side Blazor templates still have the old names for Preview 7. They have been updated for Preview 8.

Reply:
They were reverted back from these names to the prior names? ... because we changed them when readers sent screenshots of what VS was showing.

https://github.com/aspnet/AspNetCore.Docs/issues/13402#issuecomment-513330883

Reply:
@guardrex It's complicated 😝. But this is what you see with the Preview 7 bits: 

![image](https://user-images.githubusercontent.com/1874516/61751526-85ced800-ad5d-11e9-9839-3e4cfcdfef84.png)

The new names will start showing up with Preview 8.